pkgs/development/haskell-modules/hackage-packages.nix

```diff
@@ -100362,7 +100362,6 @@ self: {
        description = "GI friendly Binding to the Cairo library";
        license = stdenv.lib.licenses.lgpl21;
        hydraPlatforms = stdenv.lib.platforms.none;
```

Comment:
Ah, sorry, I should have mentioned this before, but you'll also need to remove these `hdyraPlatforms =` lines, otherwise these packages won't get built by Hydra.

Reply:
OK done.